github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/dsl/GitHubPRJobDslExtension.java

```diff
@@ -44,8 +45,20 @@ public Object commitStatusOnGH(Runnable closure) {
                 context.unstableAs(),
                 null,
                 null,
-                null
-        );
+                null);
```

Comment:
no formatter - i have an eclipse plugin that is supposed to fix whitespace issues and make sure there aren't tabs, etc. my guess is that's what happened here.

i will adjust it along side whatever method renaming i have to do.